modules/profiling-ext/report/libs/jquery-3.3.1.min.js

```diff
@@ -0,0 +1,2 @@
+/*! jQuery v3.3.1 | (c) JS Foundation and other contributors | jquery.org/license */
```

Comment:
We probably need to add license header in this file

Reply:
I have clarified the license.